pkg/cmd/cli/describe/projectstatus.go

```diff
@@ -214,18 +215,27 @@ func (d *ProjectStatusDescriber) Describe(namespace, name string) (string, error
 			fmt.Fprintln(out, "Errors:")
 			for _, marker := range errorMarkers {
 				fmt.Fprintln(out, indent+"* "+marker.Message)
+				if d.Suggest && len(marker.Suggestion) > 0 {
+					fmt.Fprintln(out, indent+marker.Suggestion.String())
```

Comment:
I'm guessing @smarterclayton wanted this here:

> Users won't know what ** means - so it's probably better to have something
like:

>* you have blah blah
  fix: change blah to blah